components/Cards/index.js

```diff
@@ -17,3 +17,62 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+axios
+.get("https://lambda-times-backend.herokuapp.com/articles")
```

Comment:
It's good practice to indent chained methods (like seen here) for easier readability